src/sentry/web/urls.py

```diff
@@ -365,7 +362,7 @@ def init_all_applications():
     ),
     url(
         r'^organizations/(?P<organization_slug>[\w_-]+)/members/$',
-        OrganizationMembersView.as_view(),
+        react_page_view,
```

Comment:
Do we explicitly declare all react routes in python as well?

Reply:
Not necessarily, here are some of the scenarios I've come across where we do:

1) There are other parts of the code base that reference django's name route. In this case, there's an email template (as well as a few other django views) that use the member's named route.

2) An example from earlier this week is if you wanted a route to `/organizations/fixed-path/` to a react view, if you don't add a django route, it will think that it's an org slug and if it's not found will never load the react view.